src/openrct2/Context.cpp

```diff
@@ -1220,6 +1221,16 @@ namespace OpenRCT2
         {
             return &_newVersionInfo;
         }
+
+        void SetTimeScale(float newScale) override
+        {
+            _timeScale = std::clamp(newScale, 0.0f, 5.0f);
```

Comment:
Seems the two values are related?
```suggestion
            _timeScale = std::clamp(newScale, 0.0f, GAME_MAX_UPDATES);
```
Also does it make sense to allow 0 scale?

Reply:
Good point, I think 0.1 should be small enough.

Reply:
GAME_MAX_UPDATES is used for something else, I accidentally changed that.